lib/matplotlib/offsetbox.py

```diff
@@ -1242,13 +1242,13 @@ def __init__(self, offsetbox, xy,
             The position *(x, y)* to place the text at. The coordinate system
             is determined by *boxcoords*.
 
-        xycoords : str or `.Artist` or `.Transform` or callable or \
-(float, float), default: 'data'
+        xycoords : single or two-tuple of str or `.Artist` or `.Transform` or \
```

Comment:
I'm wondering if this is correct? I think that it is either a single str or a tuple of two floats, so the original should be correct. Or do you think that it was incorrect?

Reply:
Looks like this was pulled in from #24643.